core-plugins/src/main/java/co/cask/hydrator/plugin/common/StructuredToOrcTransformer.java

```diff
@@ -60,7 +60,7 @@ public OrcStruct transform(StructuredRecord input, Schema schema) {
         orcRecord.setFieldValue(fields.get(i).getName(), writable);
       } catch (UnsupportedTypeException e) {
         LOG.debug("{} is not a supported type", field.getName(), e);
-        throw new IllegalArgumentException(String.format("{} is not a supported type", field.getName()), e);
+        throw new IllegalArgumentException(String.format("%s is not a supported type", field.getName()), e);
```

Comment:
is it possible to include `input`?

Reply:
The field type and name will already be in the exception message, so I think its fine.